cmd/util/ledger/migrations/storage_v5.go

```diff
@@ -418,6 +421,12 @@ func (m StorageFormatV5Migration) reencodePayload(
 		version,
 	)
 	if err != nil {
+		// If there are empty containers without type info (e.g: at root level)
+		// Then drop such values.
+		if _, ok := err.(*EmptyContainerTypeInferringError); ok {
+			return nil, nil
+		}
+
```

Comment:
is it possible these include the arts that was sell and their data basically was upload to the root?

Reply:
I think we should keep a flag for this as well, in safe mode we don't remove anything

Reply:
If we leave those, it would crash the decoder as the type info is not available. 

btw, there was nothing on the mainnet that goes in this path. So putting this behind the flag should be fine too.

Reply:
I see, then thats fine